### ISSUE TITLE
Resolve Accessibility Issues per Lighthouse

### ISSUE THREAD

**Issue by JasonFritsche** [JasonFritsche/motivationalCats#118]
Lighthouse gave us a 79% score for accessibility. This should be at 100%. Please resolve the accessibility issues.
